projects/ui/src/Context/AppUtilsContext.tsx

```diff
@@ -0,0 +1,38 @@
+import { ReactNode, createContext, useState } from "react";
+import { useEventListener } from "../Utility/utility";
+
+//
+// Types
+//
+interface AppUtilsProviderProps {
+  children?: ReactNode;
+}
+interface IAppUtilsContext extends AppUtilsProviderProps {
+  windowInnerWidth: number;
+  windowInnerHeight: number;
+}
+
+//
+// Context
+//
+export const AppUtilsContext = createContext({} as IAppUtilsContext);
+
+//
+// Provider
+//
+export const AppUtilsContextProvider = (props: AppUtilsProviderProps) => {
+  const [windowInnerWidth, setWindowInnerWidth] = useState(window.innerWidth);
+  const [windowInnerHeight, setWindowInnerHeight] = useState(
+    window.innerHeight
+  );
+  useEventListener(window, "resize", () => {
+    setWindowInnerWidth(window.innerWidth);
+    setWindowInnerHeight(window.innerHeight);
```

Comment:
I like this being exposed via a context on app level

Reply:
Yep! I brought this over from GME.